test/unit/core/connection_string.test.js

```diff
@@ -22,10 +22,7 @@ const skipTests = [
   'Unsupported option values are ignored',
 
   // We don't actually support `wtimeoutMS` which this test depends upon
-  'Deprecated (or unknown) options are ignored if replacement exists',
-
-  // We already handle this case in different ways
-  'may support deprecated gssapiServiceName option (GSSAPI)'
```

Comment:
I removed this as it was the last reference to `gssapiServiceName` and there is no test with that description anymore.